package.json

```diff
@@ -341,8 +341,15 @@
     "devDependencies": {
         "@types/mocha": "^2.2.43",
         "@types/node": "^6.0.89",
+        "del": "^3.0.0",
+        "gulp": "^3.9.1",
```

Comment:
Gulp? Why is it needed here?

Reply:
Same reason as with the gitignore.
The only way to test other languages is by packaging (according to the nls docs) and then running it with a changel locale setting.

I will provide this info in the "How the localization technically works" section too later.

Reply:
Before you ask:

There is no gulpfile yet comitted, since i am still testing the functionality.